example/template/demo.html

```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8">
     <title>Conversation Demo</title>
     <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
-    <script src="/dist/WIDGET_CODE"></script>
```

Comment:
This works fine with http://localhost:8282/example/demo, but your fix will work with the file:// protocol as well.